composer.json

```
@@ -66,8 +66,8 @@
         "monolog/monolog": "^2.3",
         "pelago/emogrifier": "^5.0.0",
         "php-amqplib/php-amqplib": "~3.0.0",
-        "phpseclib/mcrypt_compat": "1.0.8",
-        "phpseclib/phpseclib": "2.0.*",
+        "phpseclib/mcrypt_compat": "^2.0",
```

I would not recommend using such a broad constraint for the security library, please update it to
```suggestion
        "phpseclib/mcrypt_compat": "~2.0.0",
```
The same about the dependency below

---

Done!